rllib/evaluation/rollout_worker.py

```diff
@@ -552,6 +552,10 @@ def gen_rollouts():
         self.total_rollout_fragment_length: int = (
             configured_rollout_fragment_length * self.config.num_envs_per_worker
         )
+        # Remember if we are deailing with an Atari env.
+        # TODO(jungong) : is_atari and preprocessing_enabled should probably be
+        # available on Policy instances as well.
+        self.is_atari = False
         self.preprocessing_enabled: bool = not config._disable_preprocessor_api
```

Comment:
see here we have `self.preprocessing_enabled = not config._disable_preprocessor_api` so I don't understand why we have two different flags here?